browser/ui/views/sidebar/sidebar_items_scroll_view.cc

```diff
@@ -51,6 +51,7 @@ class SidebarItemsArrowView : public views::ImageButton {
     DCHECK(GetInstallFocusRingOnFocus());
     views::FocusRing::Get(this)->SetColorId(gfx::kBraveBlurple300);
     SetAccessibleName(accessible_name);
+    SetPaintToLayer();
```

Comment:
need this because of item's ink drop?

Reply:
Yeah, otherwise ink drop will be visible on top of the arrow button